ui/app/AppLayouts/Chat/ChatColumn/ChatComponents/SignTransactionModal.qml

```diff
@@ -17,26 +17,15 @@ ModalPopup {
     property alias transactionSigner: transactionSigner
 
     property var sendTransaction: function(selectedGasLimit, selectedGasPrice, enteredPassword) {
-        let responseStr = walletModel.sendTransaction(root.selectedAccount.address,
-                                                 root.selectedRecipient.address,
+        let responseStr = walletModel.sendTransaction(selectFromAccount.selectedAccount.address,
+                                                 selectRecipient.selectedRecipient.address,
                                                  root.selectedAsset.address,
                                                  root.selectedAmount,
                                                  selectedGasLimit,
                                                  selectedGasPrice,
                                                  enteredPassword,
                                                  stack.uuid)
 
-        let response = JSON.parse(responseStr)
-
-        if (response.error) {
-            if (response.result.includes("could not decrypt key with given password")){
-                //% "Wrong password"
-                transactionSigner.validationError = qsTrId("wrong-password")
-                return
-            }
-            sendingError.text = response.result
-            return sendingError.open()
-        }
```

Comment:
This error handling here is not functioning because `wallet.sendTransaction` doesn't return anything since it's asynchronous. It resolves later and the same error handling is already done in its callback. We can safely remove this code